Lib/test/test_mailcap.py

```diff
@@ -242,17 +242,20 @@ def test_test(self):
         self._run_cases(cases)
 
     def test_unsafe_mailcap_input(self):
-        c     = MAILCAPDICT
         plist = ["total=*"]
 
-        with self.assertWarnsRegex(mailcap.UnsafeMailcapInput,'Refusing to substitute parameter.*into a shell command'):
+        with self.assertWarnsRegex(mailcap.UnsafeMailcapInput,
+                                   'Refusing to substitute parameter.*into a shell command'):
             self.assertEqual(mailcap.subst("echo %{total}", "audio/wav", "foo.txt", plist), None)
-        
-        with self.assertWarnsRegex(mailcap.UnsafeMailcapInput,'Refusing to substitute MIME type.*into a shell'):
+
+        with self.assertWarnsRegex(mailcap.UnsafeMailcapInput,
+                                   'Refusing to substitute MIME type.*into a shell'):
             self.assertEqual(mailcap.subst("echo %t", "audio/*", "foo.txt"), None)
-        
-        with self.assertWarnsRegex(mailcap.UnsafeMailcapInput,'Refusing to use mailcap with filename.*Use a safe temporary filename.'):
-            self.assertEqual(mailcap.findmatch(c, "audio/wav", filename="foo*.txt"), (None, None))
+
+        with self.assertWarnsRegex(mailcap.UnsafeMailcapInput,
+                                   'Refusing to use mailcap with filename.*Use a safe temporary filename.'):
+            self.assertEqual(mailcap.findmatch(MAILCAPDICT,
+                                               "audio/wav", filename="foo*.txt"), (None, None))
```

Comment:
We don't want to break the first arg into two lines and then continue with the second arg on the same line.

You can assign the return value of findmatch to a variable and then compare the variable to expected value. 
Otherwise, add a newline before the (None, None) and indent it like "mailcap.findmatch".